src/main/java/pingmebot/PingMeException.java

```diff
@@ -0,0 +1,17 @@
+package pingmebot;
+
+/**
+ * Responsible for throwing exception that is specific to the bot application.
+ */
+public class PingMeException extends Exception {
```

Comment:
Can consider extending PingMeException to encapsulate the different kinds of errors your application runs into, so you have an easier time if you want to handle them differently.